docs/sphinx/install.rst

```diff
@@ -103,6 +103,25 @@ or run the Python examples using the Python interpreter.
     may not be automatically active in the container environment. You may need to install your preferred 
     extension in the container environment for all of your development tools to be available.
 
+Python wheels
+--------------------
+
+CUDA Quantum Python wheels are available on [PyPI.org](https://pypi.org/project/cuda-quantum). 
+The CUDA Quantum Python wheels contain the Python API and core components of
+CUDA Quantum. For more information about available packages and documentation,
+see :doc:`versions`.
+
+To install the latest release using `pip <https://pypi.org/project/pip/>`__, run
+
+.. code-block:: console
+
+    pip install cuda-quantum
```

Comment:
I usually advise `python3 -m pip install ...` since users often have more than one `python` / `pip` executable sitting on the system, and sometimes it's version 2.

Reply:
done